examples/02_decoding/plot_haxby_searchlight_surface.py

```diff
@@ -4,6 +4,16 @@
 
 This is a demo for surface-based searchlight decoding,
 as described in :footcite:t:`Chen2011`.
+
+.. warning::
+
+    This example projects results from the Haxby dataset
+    on the fsaverage surface.
+    This is inappropriate
+    given that the Haxby data has not been properly coregistered
+    to allow for such projection.
+    It is done in this example for pedagogical reasons
+    to show "how to do it".
```

Comment:
As discussed in dev meeting, we need to replace the Haxby dataset with properly coregistered dataset for this example.

(Could not find the issue where this was initially discussed hence re-posting it here to further ref in a new issue)

Reply:
- relates to https://github.com/nilearn/nilearn/issues/4561